client/src/commons/NumericTextField/NumericTextField.tsx

```diff
@@ -8,9 +8,9 @@ const stringAlphabet = yup
   .string()
   .required()
   .matches(/^[0-9]*$/)
-  .max(50);
+  .max(10);
 
-const errorMessage = 'השדה יכול להכיל רק מספרים';
+const errorMessage = 'ערך לא תקין';
```

Comment:
why did you replece the "only numbers" message?

Reply:
זה לא רק מספרים זה גם אורך מעל 10 - "השדה יכול להכיל רק מספרים ולא יותר מ10" היה נראה לי ארוך מדי

Reply:
"השדה יכול להכיל עד 10 מספרים" אולי?

Reply:
ההודעה הזאת לא מושלמת וגם זאת לא מושלמת, אני חושב שבכל מקרה השדה יכול להכיל רק מספרים נותן יותר אינפורמציה מאשר ערך לא תקין
במקומות בהן יש גם ולידציה על מספר טלפון אז תנתן הודעת שגיאה מותאמת אישית

Reply:
השדה מכיל עד עשר ספרות בלבד
מספר טלפון מכיל עד עשר ספרות

אנסה לחשוב על עוד

Reply:
הלכתי על של אמילי

Reply:
לאחר בדיקה זו אותה הודעה שתופיע בשגיאה של שדה ת.ז, צריך לשנות את הודעת השגיאה או להכניס הודעת שגיאה בPROPS